packages/network/src/json-rpc-provider.ts

```diff
@@ -26,9 +26,9 @@ export interface JsonRpcProviderOptions {
 
 // JsonRpcProvider with a middleware stack. By default it will use a simple caching middleware.
 export class JsonRpcProvider extends ethers.JsonRpcProvider implements EIP1193Provider, JsonRpcSender {
-  #chainId?: number
-  #nextId: number = 1
-  #sender: EIP1193Provider
+  private chainId?: number
```

Comment:
Was this giving any trouble somewhere else? I am not sure if using real private fields was needed in the first place, or just someone playing with new features

Reply:
I was having compile errors. Probably an issue on my end but didn't see the reason to keep it when we don't follow this pattern anywhere else.

Reply:
This change was made by @corbanbrook. Let’s have him chime in, just in case there was a specific need for it.

Reply:
I spoke with Corban, and he doesn’t recall any specific reason for using the private fields. It’s okay to proceed with reverting the change to using `private`.